indico/modules/categories/client/js/management.js

```diff
@@ -435,4 +435,55 @@
       },
     });
   }
+
+  function setupRolesToggle() {
+    var $roles = $('#event-roles');
+    $roles.on('click', '.toggle-members', function() {
+      var $row = $(this)
+        .closest('tr')
+        .next('tr')
+        .find('.slide');
+      $row.css('max-height', $row[0].scrollHeight + 'px');
+      $row.toggleClass('open close');
+    });
+
+    $roles.on('indico:htmlUpdated', function() {
+      $(this)
+        .find('.slide')
+        .each(function() {
+          $(this).css('max-height', this.scrollHeight + 'px');
+        });
+    });
+  }
+
+  function setupRolesButtons() {
+    $('#event-roles').on('click', '.js-add-members', function(evt) {
+      var $this = $(this);
+      evt.stopPropagation();
```

Comment:
It is usually the first thing to appear in the callback (however there is nothing wrong with putting it after `var $this`)

Reply:
I didn't get this one.